static/app/views/detectors/components/detectorDetailsSidebar.tsx

```diff
@@ -0,0 +1,246 @@
+import Placeholder from 'sentry/components/placeholder';
+import TextOverflow from 'sentry/components/textOverflow';
+import TimeSince from 'sentry/components/timeSince';
+import Section from 'sentry/components/workflowEngine/ui/section';
+import {IconArrow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {PriorityLevel} from 'sentry/types/group';
+import {
+  DataConditionType,
+  DetectorPriorityLevel,
+} from 'sentry/types/workflowEngine/dataConditions';
+import type {Detector} from 'sentry/types/workflowEngine/detectors';
+import {useTeamsById} from 'sentry/utils/useTeamsById';
+import useUserFromId from 'sentry/utils/useUserFromId';
+import DetailsPanel from 'sentry/views/detectors/components/detailsPanel';
+import {getResolutionDescription} from 'sentry/views/detectors/utils/getDetectorResolutionDescription';
+
+const DETECTOR_PRIORITY_LEVEL_TO_PRIORITY_LEVEL: Record<
+  DetectorPriorityLevel,
+  PriorityLevel
+> = {
+  [DetectorPriorityLevel.LOW]: PriorityLevel.LOW,
+  [DetectorPriorityLevel.MEDIUM]: PriorityLevel.MEDIUM,
+  [DetectorPriorityLevel.HIGH]: PriorityLevel.HIGH,
+  [DetectorPriorityLevel.OK]: PriorityLevel.LOW,
+};
+
+function getDetectorEnvironment(detector: Detector) {
+  return (
+    detector.dataSources?.find(ds => ds.type === 'snuba_query_subscription')?.queryObj
+      ?.snubaQuery.environment ?? t('All environments')
+  );
+}
+
+function AssignToTeam({teamId}: {teamId: string}) {
+  const {teams} = useTeamsById({ids: [teamId]});
+  const team = teams.find(tm => tm.id === teamId);
+  return t('Assign to %s', `#${team?.slug ?? 'unknown'}`);
+}
+
+function AssignToUser({userId}: {userId: string}) {
+  const {isPending, data: user} = useUserFromId({id: parseInt(userId, 10)});
+
+  if (isPending) {
+    return (
+      <Flex align="center" gap={space(0.5)}>
+        {t('Assign to')} <Placeholder width="80px" height="16px" />
+      </Flex>
+    );
+  }
+
+  const title = user?.name ?? user?.email ?? t('Unknown user');
+  return (
+    <Tooltip title={title} showOnlyOnOverflow>
+      {t('Assign to %s', title)}
+    </Tooltip>
+  );
+}
+
+function DetectorPriorities({detector}: {detector: Detector}) {
+  const detectionType = detector.config?.detection_type || 'static';
+
+  // For dynamic detectors, show the automatic priority message
+  if (detectionType === 'dynamic') {
+    return <div>{t('Sentry will automatically update priority.')}</div>;
+  }
+
+  const conditions = detector.conditionGroup?.conditions || [];
+
+  // Filter out OK conditions and sort by priority level
+  const priorityConditions = conditions
+    .filter(condition => condition.conditionResult !== DetectorPriorityLevel.OK)
+    .sort((a, b) => (a.conditionResult || 0) - (b.conditionResult || 0));
+
+  if (priorityConditions.length === 0) {
+    return <div>{t('No priority thresholds configured')}</div>;
+  }
+
+  const getConditionLabel = (condition: (typeof priorityConditions)[0]) => {
```

Comment:
This is similar to what I have in DetectorLink: https://github.com/getsentry/sentry/blob/55b9743c8b95cc8b6d96578a01eb01a8edae7c38/static/app/views/detectors/components/detectorLink.tsx#L57

We don't have to do it yet, but it might be good to start thinking about whether we can share any logic or colocate the formatting logic at all